GetIntoTeachingApi/Services/MetricService.cs

```diff
@@ -20,12 +20,12 @@ public class MetricService : IMetricService
         private static readonly Histogram _teachingEventSearchResults = Metrics
             .CreateHistogram("api_teaching_event_search_results_count", "Histogram of teaching event search results.", new HistogramConfiguration
             {
-                LabelNames = new[] { "type_id", "radius" },
+                LabelNames = new[] { "type_ids", "radius" },

```

Comment:
We'll need to update some of the Grafana panels once this changes (probably just on the search dashboard)